app/routes.js

```diff
@@ -0,0 +1,7 @@
+// const controller = require('./controllers/controller');
+
+exports.init = app => {
+  // app.get('/endpoint/get/path', [], controller.methodGET);
```

Comment:
In general, done comments its a bad practice but for the moment its fine

Reply:
Perfect! in the next pr i will have this in mind! 👍